Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -444,13 +444,13 @@ class SendViewController: UIViewController {
         self.resetViews()
         
         SendCompletedViewController.showSendCompletedDialog(for: txHash) { showTxDetails in
-            let slideMenuController = self.slideMenuController()!
-            (slideMenuController.leftViewController as! NavigationMenuViewController).changeActivePage(to: .overview)
+//            let slideMenuController = self.slideMenuController()!
```

Comment:
Is there no way to view a tx details page from here? How is it done from the history page?

Reply:
this is the same implementation as the history page. i have changed the `modalPresentationStyle` to `.overCurrentContext` so it just pushes on top of the send page. what are your thoughts on that